app/javascript/components/common/TrackIcon.css

```diff
@@ -0,0 +1,5 @@
+:root {
+    --c-track-icon-width-ratio: 92.5%;
```

Comment:
This goes with:
```
            width: calc(var(--c-track-icon-width-ratio) * var(--nav-height));
```

...to allow us to pre-specify image sizes via `calc` and a known `height`. This means we can specify just the height, and the page will allow the right space for the image (maintaining a correct aspect ratio), which avoids the page jumping around once images actually load.